File: Getting-Practice-with-Python-and-Javascript/4- Functions/Javascript/Excercises/program-7.js
// 7.  Escribe una función que tome un número y devuelva su factorial.


let factorialOf = (num) => {
    let contador = num;
    let lista = [];
    for (let i = 1; i < num; i++) {
        contador *= i; // operacion factorial
        lista.push(i); // lista de numeros que son factorial de num
    }

    console.log(lista);
    return contador;
}

console.log(factorialOf(5))

File: Getting-Practice-with-Python-and-Javascript/4- Functions/Python/Excercises/program-7.py
# 7.  Escribe una función que tome un número y devuelva su factorial.


def factorialOf(num):
    listaFactorial = []
    contador = num
    for c in range(1, num):
        # lista de numeros que seran parte de la factorial de num
        listaFactorial.append(c)
        # operacion para conseguir factorial de num
        contador *= c
    print(listaFactorial)
    return contador


print(factorialOf(5))
